src/index.html

```diff
@@ -1,13 +1,378 @@
+              <div class="grid">
+                <div class="
+                  grid__item
+                  grid__item--tablet-1-2
+                  grid__item--desktop-1-4
+                  "
+                >
+                  <div class="detail__photo-container">
+                    <img
+                    src="./images/photos/3_1.jpg"
+                    alt="Detail 1"
+                    class="detail__photo"
+                    >
+                  </div>
+                </div>
+                <div class="
+                  grid__item
+                  grid__item--tablet-3-6
+                  grid__item--desktop-5-12
+                  "
+                >
+                  <div class="detail__photo-container">
+                    <img
+                    src="./images/photos/3_2.jpg"
+                    alt="Detail 2"
+                    class="detail__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <h3 class="detail__title">Hydraulic disc brakes & Lightweight</h3>
+            <div class="detail__description">
+              Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </div>
+          </div>
+
+          <div class="grid grid--tablet">
+            <div class="
+              grid__item
+              grid__item--tablet-2-5
+              grid__item--desktop-5-8
+              "
+            >
+              <a href="/" class="button">
+                Explore
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contacs" id="contacs">
+        <div class="container">
+          <h2 class="page__section-title">Contact us</h2>
+          <div class="grid grid--tablet">
+            <div class="grid__item grid__item--tablet-1-3 grid__item--desktop-1-5">
+              <form class="form" action="#">
+                <input
+                  name="name"
+                  placeholder="Name"
+                  type="text"
+                  class="contacs__field form-field"
+                  required
+                >
+
+                <input
+                  name="email"
+                  placeholder="Email"
+                  type="email"
+                  class="contacs__field form-field"
+                  required
+                >
+
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="
+                    contacs__field
+                    contacs__field--last
+                    form-field
+                    form-field--textarea"
+                    required
+               ></textarea>
+
+                <button class="button button--contacs">Send</button>
```

Comment:
button should be `type="submit"` and then implement these checklist points
<img width="797" alt="image" src="https://user-images.githubusercontent.com/102411130/211506268-10c3ac99-77fa-459d-b748-7e40536fb555.png">

Reply:
> button should be `type="submit"` and then implement these checklist points <img alt="image" width="797" src="https://user-images.githubusercontent.com/102411130/211506268-10c3ac99-77fa-459d-b748-7e40536fb555.png">

sorry i dont push my fixed